subprojects/junit5/junit5.gradle

```diff
@@ -0,0 +1,14 @@
+description = "Mockito JUnit 5 support"
+
+apply from: "$rootDir/gradle/java-library.gradle"
+
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
+
+dependencies {
+    compile project.rootProject
+    testRuntime libraries.assertj
```

Comment:
I think you should use `testCompile` or `testImplementation` instead of `testRuntime`.
Otherwise, AssertJ classes would not be read at build time and you would not be able to compile test classes.